packages/validator/src/commands/convertCmd.ts

```diff
@@ -1,33 +1,18 @@
-import fs from 'fs';
 import yargs from 'yargs';
 import { hideBin } from 'yargs/helpers';
 import * as utils from './utils';
-import { convert, convertJson } from '../convertor';
-import { Log, Result } from '../types';
-
-const oas2ois = 'OAS2OIS.json';
-const ois2config = 'OIS2Config.json';
-
-type FromChoices = 'oas' | 'ois';
-const fromChoices: ReadonlyArray<FromChoices> = ['oas', 'ois'];
-type ToChoices = 'ois' | 'config';
-const toChoices: ReadonlyArray<ToChoices> = ['ois', 'config'];
-
-const supportedConversions = [
-  ['oas', 'ois'],
-  ['oas', 'config'],
-  ['ois', 'config'],
-];
+import { convert } from '../convertor';
+import { Log } from '../types';
 
 const args = yargs(hideBin(process.argv))
```

Comment:
It would be nice if we could add something like `--list-conversions` listing all the available conversions (including versions). That's just something nice to have for the future.